src/hooks/hooks/handlers/BigOrderIsProfitHookHandler.ts

```diff
@@ -0,0 +1,139 @@
+import BigNumber from "bignumber.js";
+import Web3 from "web3";
+
+import { createClientLogger } from "../../../logger";
+import { Notification } from "../../notification/Notification";
+import { TwitterNotification } from "../../notification/TwitterNotification";
+import { OrderEstimatedParams } from "../../types/params/OrderEstimatedParams";
+import { Hook } from "../Hook";
+
+export const hookHandlerBigOrderIsProfit = (
```

Comment:
`hookHandler` prefix is useless. Just 'bigOrderIsProfit'

Add hooks/index.ts to export all hookHandlers at once

Better naming: `announceLargeOrder`